kintone.d.ts

```diff
@@ -255,4 +255,175 @@ declare namespace kintone {
 
     function getLoginUser(): LoginUser;
     function getUiVersion(): 1 | 2;
+
+    namespace types.fields {
```

Comment:
@yokotaso 
`dts-gen` generates a type definition for an app under the namespace of `kintone.types.Fields` even though we can change the namespace by `--namespace` option.
So this namespace(`kintnoe.types.fields`) might be confusing.

- `kintone.types.fields`
- `kintone.types.Fields`

Is there any other namespace for this?